src/main/resources/screens/POGR_BUS_CONTRIBUTIONS_ADD.json

```diff
@@ -8,12 +8,35 @@
         "required"
       ],
       "props": {
-        "choices": "POGR_CONTRIB_BUS_AREA"
+        "choices": "POGR_CONTRIB_BUS_AREA",
```

Comment:
Would this be better using the conditionalChoices mechanic here than introducing a whole new field?

Reply:
Especially as this is the only thing changing?

Reply:
Apart from this - name and label shall be included as well. Do we really need to using condition to render all 3 properties (choices, name and label) ?

Reply:
No you just need to change the choices that are shown to the user.

Reply:
I have incorporated your comment and it makes code tidy as well 👍